robot/EDA/resources/EDA.py

```diff
@@ -418,3 +436,33 @@ def verify_dropdown_field_status(self, **kwargs):
             expected_value = bool(expected_value == "disabled")
             if not str(expected_value).lower() == str(actual_value).lower() :
                 raise Exception (f"Drop down field {field} status is {actual_value} instead of {expected_value}")
+
+    def verify_checkbox_value(self,**kwargs):
+        """ This method validates the checkbox value for the field passed in kwargs
+            Pass the field name and expected value to be verified from the tests using
+            keyword arguments. true - checked, false - unchecked
+        """
+        self.selenium.execute_javascript("window.scrollTo(0, document.body.scrollHeight)")
+        self.selenium.execute_javascript("window.scrollTo(document.body.scrollHeight, 0)")
+        for field,expected_value in kwargs.items():
+            locator = eda_lex_locators["eda_settings_system"]["default_checkbox"].format(field)
+            self.selenium.page_should_contain_element(locator)
+            self.selenium.wait_until_element_is_visible(locator,
+                                                error= "Element is not displayed for the user")
+            actual_value = self.selenium.get_element_attribute(locator, "alt")
+            if not str(expected_value).lower() == str(actual_value).lower() :
+                raise Exception (f"Checkbox value in {field} is {actual_value} but it should be {expected_value}")
+
+    def verify_dropdown_value(self,**kwargs):
+        """ This method validates the dropdown value for the field passed in kwargs
+            Pass the field name and expected value to be verified from the tests using
+            keyword arguments
+        """
+        for field,expected_value in kwargs.items():
+            locator = eda_lex_locators["eda_settings_system"]["default_dropdown_value"].format(field,expected_value)
+            self.selenium.page_should_contain_element(locator)
+            self.selenium.wait_until_element_is_visible(locator,
```

Comment:
same here

Reply:
This step was added to eliminate inconsistent failures